frontend/src/wizards/Argo/MultipleSourcesSelector.tsx

```diff
@@ -58,7 +66,7 @@ export function MultipleSourcesSelector(props: MultipleSourcesSelectorProps) {
       <WizHidden hidden={(data) => data.repositoryType}>
         <Title headingLevel="h6">{t('Repository type')}</Title>
       </WizHidden>
-      <WizTiles path="repositoryType">
+      <WizTiles path="repositoryType" required>
```

Comment:
Oh wow! This worked. I assumed you would need to implement this support, but glad to see it's already there.